modules/aws_sso_permission_set/README.md

```diff
@@ -0,0 +1,148 @@
+# AWS SSO Permission Set
```

Comment:
Shall we use AWS IAM Identity Center Permission Set, and also use this as default in the text, we can than say (formerly known as AWS SSO). Lets use the new name by default instead of the other way around